sabnzbd/postproc.py

```diff
@@ -230,6 +230,18 @@ def run(self):
         else:
             logging.debug("Completed Download Folder %s is not on FAT", complete_dir)
 
+        def check_directory_writing_capability(dir):
```

Comment:
I don't like this function-inside-a-function (`run(self)`). Please put it with the the other functions, or in `sabnzbd.filesystem`.